hotel-management/src/pages/CartPage.tsx

```diff
@@ -0,0 +1,5 @@
+const CartPage = () => {
```

Comment:
Use shorthand recheck for all

Reply:
and also export inline. If in the future u have to rename the component then u must rename two place